frontend/src/lib/utils.test.ts

```diff
@@ -271,23 +271,23 @@ describe('dateFilterToText()', () => {
             const from = dayjs('2018-04-04T16:00:00.000Z')
             const to = dayjs('2018-04-09T15:05:00.000Z')
 
-            expect(dateFilterToText(from, to, 'custom', dateMapping, true)).toEqual('4 Apr 2018 - 9 Apr 2018')
+            expect(dateFilterToText(from, to, 'custom', dateMapping, true)).toEqual('April 4 - April 9, 2018')
```

Comment:
If the year changes (e.g. april 4 2016 - april 9 2018), will it show both years?

Reply:
It does and I added a test